demos/TicTacToe/readme.md

```diff
@@ -0,0 +1,71 @@
+Firebase Cocos2d-x TicTacToe Sample 
+==========================
+
+Desktop cocos2d-x sample using Firebase Real-Time Database and Firebase Authentication.
+
+Introduction
+------------
+
+- [Read more about Firebase](https://firebase.google.com/docs)
+
+Prerequisites
+-------------
+- [Cmake](https://cmake.org/download/) (Minimum 3.6)
+
+- [Python 2.7](https://www.python.org/download/releases/2.7/) 
+
+- The latest version [of Cocos2d-x](https://cocos2d-x.org/download) (cocos environment variable must be added to PATH) 
+
+-  [Firebase CPP SDK](https://cocos2d-x.org/download) (Must be add to path under FIREBASE_CPP_SDK_DIR) 
+
+-  Windows: ([Visual Studio 2019](https://visualstudio.microsoft.com/downloads/))
```

Comment:
Out of curiosity, are there modern C++ features we're using that necessitate VS2019? 

(You don't have to go through the trouble of finding the earliest compatible version right now, I'm just curious)

Reply:
The cocos cmake guide calls defaults to visual studio 15 2017 so it does not require 2019. I would have to research and/or experiment to see the minimum visual studio generator required. I plan on making this a flag in the future.